Sources/Watch.swift

```diff
@@ -1008,7 +1018,7 @@ fileprivate extension Watch {
           "propertySet": propertySet,
         ]
       
-      return ([dictionary] as NSArray).filtered(using: predicate.predicate).isEmpty == false
+      return predicate.predicate.evaluate(with: dictionary)
     }
     
     let namedOnesThatPass = objects.filter {
```

Comment:
Let's eliminate this variable and pass the `object.filter` call to the `nodesThatPass.union` call directly.